src/Features/VisualBasic/Portable/AddFileBanner/VisualBasicAddFileBannerCodeRefactoringProvider.vb

```diff
@@ -5,7 +5,8 @@ Imports Microsoft.CodeAnalysis.AddFileBanner
 Imports Microsoft.CodeAnalysis.CodeRefactorings
 
 Namespace Microsoft.CodeAnalysis.VisualBasic.AddFileBanner
-    <ExportCodeRefactoringProvider(LanguageNames.VisualBasic), [Shared]>
+    <ExportCodeRefactoringProvider(LanguageNames.VisualBasic,
+        Name := PredefinedCodeRefactoringProviderNames.AddFileBanner), [Shared]>
```

Comment:
Nit: this formatting is likely wrong (VB defaults to no space before or after `:=`). Verify that formatting the change leaves it as-is.

Reply:
I just blatantly copy pasted the formatting from existing code.